### ISSUE TITLE
The compilation of ViDeZZo-qemu encountered an error

### ISSUE THREAD

**Issue by Ashieta** [HexHive/ViDeZZo#7]
Hello，when I followed the Quick Start guide to build the container and compile ViDeZZo-qemu, I encountered this error：
```
...
+ ../configure --enable-videzzo --enable-fuzzing --enable-debug --disable-werror --enable-sanitizers --enable-spice '--target-list=i386-softmmu x86_64-softmmu arm-softmmu aarch64-softmmu'
ERROR: unknown option --enable-videzzo
...
```
Specifically, this error occurs in the execution of the Makefile statement `bash -x 0002-copy-to-qemu.sh` in videzzo_qemu/Makefile, line 20.
By the way, I found that the `videzzo_qemu/qemu/configure` file is exactly the same as the original configure file of `qemu`, which may explain that `--enable-videzzo `cannot be used. Hope to get your help :)

**Comment by cyruscyliu** [comment]
Hi, this should be fixed in 04a2a447a9b2eae45ce31ae8900f70dfefda834d. Please pull, delete `videzzo_qemu/qemu` and retry.

**Comment by Ashieta** [comment]
Now ViDeZZo can be built and run normally, thanks for your attention :)
